bin/alias.bat

```diff
@@ -14,18 +20,28 @@ if not ["%_temp%"] == ["%_temp2%"] (
 	goto:eof
 )
 
-echo %* >> "%CMDER_ROOT%\config\aliases"
-doskey /macrofile="%CMDER_ROOT%\config\aliases"
-perl "%CMDER_ROOT%\scripts\clean_aliases.pl"
-echo Alias created
+:: replace already defined alias
+findstr /b /v /i "%_temp%=" "%ALIASES%" >> "%ALIASES%.tmp"
+echo %* >> "%ALIASES%.tmp" && type "%ALIASES%.tmp" > "%ALIASES%" & @del /f /q "%ALIASES%.tmp"
+doskey /macrofile="%ALIASES%"
 endlocal
 goto:eof
 
+:p_del
+findstr /b /v /i "%2=" "%ALIASES%" >> "%ALIASES%.tmp"
+type "%ALIASES%".tmp > "%ALIASES%" & @del /f /q "%ALIASES%.tmp"
+doskey /macrofile=%ALIASES%
+goto:eof
+
 :p_reload
-doskey /macrofile="%CMDER_ROOT%\config\aliases"
+doskey /macrofile="%ALIASES%"
 echo Aliases reloaded
 goto:eof
 
+:p_show
+type "%ALIASES%" || echo No aliases found at "%ALIASES%"
+goto :eof
+
 :p_help
 echo.Usage:
 echo.	alias name=full command
```

Comment:
Agree. Done in https://github.com/vladimir-kotikov/cmder/commit/dba00b9175c5e90d7a28d562d8dc117f7516c46f